packages/aztec.js/test/initDatabase.js

```diff
@@ -0,0 +1,65 @@
+/**
```

Comment:
Can we add some printing when this file is loaded which says that we are doing this?

Reply:
Good idea, will do